allennlp/service/predictors/semantic_role_labeler.py

```diff
@@ -16,24 +16,51 @@ def __init__(self, model: Model,
 
         self.nlp = spacy.load('en', parser=False, vectors=False, entity=False)
 
+    def make_srl_string(words: List[str], tags: List[str]) -> str:
```

Comment:
Suggestions appreciated--I'm not sure the best way to code this up in python.

Reply:
no obviously better way jumps out at me